src/components/Time/style.less

```diff
@@ -2,5 +2,5 @@
   display: inline-block;
   padding: 3px;
   font-size: 12px;
-  color: var(--gray-3);
+  color: #2d2d2d;
```

Comment:
Instead of directly changing the color, do we need to update the value of the `--gray-3` variable like we did it for the element focus color. @akhilkumar-isc

Reply:
Will fix this in the next iteration. @akhilkumar-isc